sdk/mediaservices/Azure.Media.Analytics.Edge/src/Customization/MediaGraphTopologySetRequest.cs

```diff
@@ -7,20 +7,18 @@ namespace Azure.Media.Analytics.Edge.Models
 {
     public partial class MediaGraphTopologySetRequest
     {
-        internal MediaGraphTopologySetRequestBody GraphBody;
+        private MediaGraphTopologySetRequestBody _graphBody;
```

Comment:
looks like this field can be removed as well

Reply:
Fixed